google/ggrc-core
# GGRC-1991 FE: add the possibility to filter with query <=, >=

# Issue description

Add the possibility to filter with query <=, >=

# Steps to test the changes

1. Go to a tree view of controls.
2. Set "last updated" column to show.
3. Type "last updated" >= or <= "04/28/2017" (note: the date entered should match any of last updated dates in the list of controls)

**Expected result:** the controls that match this last updated date are shown.
**Actual result:** no controls are shown.

# Sanity checklist




# PR Review checklist
